zypp-curl/parser/metalinkparser.cc

```diff
@@ -431,6 +442,11 @@ MetaLinkParser::parse(const InputStream &is)
       parseBytes(buf, is.stream().gcount());
     }
   parseEnd();
+  MIL << "Parsed " << pd->urls.size() << " mirrors from " << is.path() << std::endl;
+  if ( env::ZYPP_METALINK_DEBUG() ) {
+    for ( const auto &mirr : pd->urls )
+      DBG << "- " << mirr.priority << " " << mirr.url << std::endl;
+  }
```

Comment:
Should we add the same in libzypp/zypp-curl/ng/network/private/downloaderstates/preparemulti_p.cc:122 ?

Reply:
The parser itself issues the messages. As long as you didn't implement your own MetaLinkParser it shoudl do.

Reply:
Then we also should add it to the ZSyncParser